### PR TITLE
add NPS nodein narcotice

### PR DESCRIPTION
**Pull request by Sulstice**

### CODE REVIEW

File: global_chem/global_chem/narcotics/new_psychoactive_substances.py
Comment (Sulstice) [inline global_chem/global_chem/narcotics/new_psychoactive_substances.py:1]: Make `european_union_psychoactive_substances_drug_report` the name of the python file. Can you actually name a node `European Union` and a subnode for psychoactive substance drug report.

Comment (thedevp) [reply global_chem/global_chem/narcotics/new_psychoactive_substances.py:1]: @Sulstice I'm not sure why the node is still not working. Can you have a look when you have time? thank you

File: .github/workflows/continous_integration.yml
Comment (ANUGAMAGE) [inline .github/workflows/continous_integration.yml:1]: GlobalChem API---- AttributeError: type object 'Mango' has no attribute 'get_smarts'
pre-commit----No hook with id `flake8` in stage `pre-commit`
Error: The process '/opt/hostedtoolcache/Python/3.12.2/x64/bin/pre-commit' failed with exit code 1

**Review by ANUGAMAGE — COMMENTED** [review]
GlobalChem API---- AttributeError

**Pull request merged**